.github/workflows/pr.yml

```
@@ -415,3 +415,44 @@ jobs:
         with:
           filePath: /src/summary.md
           comment_tag: execution
+
+  docker-image-push:
+    name: Docker Image push to cedana-helper-test
+    runs-on: ubicloud-standard-8
```

hmm.. do we need 8 or would 2 be enough, I don't think build performance will be affected much, could you test it.

```suggestion
    runs-on: ubicloud-standard-2
```